ansible/roles/ocp4-workload-openbanking-sandbox/tasks/post_workload.yml

```diff
@@ -0,0 +1,335 @@
+
+# # - debug:
+# #     msg: "CONTENT ---> {{ _keycloak_admin_do_auth.content | from_json }}"
+
+# - set_fact:
+#     _keycloak_admin_token: "{{ _keycloak_admin_do_auth.content | from_json | json_query('access_token') }}"
+
+# # - debug:
+# #     msg: "TOKEN ---> {{ _keycloak_admin_token }}"
+
+# - name: Listing Keycloak realm clients
+#   uri:
+#     url: "{{ _keycloak_url }}/auth/admin/realms/microcks/clients"
+#     validate_certs: "{{ obsandbox_validate_certs|d(false)|bool }}"
+#     method: GET
+#     headers:
+#       Content-Type: application/json
+#       Authorization: "Bearer {{ _keycloak_admin_token }}"
+#     return_content: yes
+#     status_code: 200
+#   register: _keycloak_get_realm_clients
+#   retries: 3
+#   delay: 5
+#   until: _keycloak_get_realm_clients.content is defined
+
+# # - debug:
+# #     msg: "REALM CLIENTS ---> {{ _keycloak_get_realm_clients }}"
+
+# - set_fact:
+#     _microcks_client_id: "{{ _keycloak_get_realm_clients.content | from_json | json_query('[?clientId == `microcks-serviceaccount`].id') | join }}"
+#   # vars:
+#   #   _query: >-
+
+# - name: Retrieve the Microcks service account client secret from Keycloak
+#   uri:
+#     url: "{{ _keycloak_url }}/auth/admin/realms/microcks/clients/{{ _microcks_client_id }}/client-secret"
+#     validate_certs: "{{ obsandbox_validate_certs|d(false)|bool }}"
+#     method: GET
+#     headers:
+#       Content-Type: application/json
+#       Authorization: "Bearer {{ _keycloak_admin_token }}"
+#     return_content: yes
+#     status_code: 200
+#   register: _keycloak_get_client_secret
+#   retries: 3
+#   delay: 5
+#   until: _keycloak_get_client_secret.content is defined
+
+# # - debug:
+# #     msg: "CLIENT SECRET ---> {{ _keycloak_get_client_secret }}"
+
+# - set_fact:
+#     _microcks_client_secret: "{{ _keycloak_get_client_secret.content | from_json | json_query('value') }}"
+
+# # - debug:
+# #     msg: "MICROCKS CLIENT SECRET ---> {{ _microcks_client_secret }}"
+
+# - name: Authenticating on Microcks realm with service account via OAuth2
+#   uri:
+#     url: "{{ _keycloak_url }}/auth/realms/microcks/protocol/openid-connect/token"
+#     validate_certs: "{{ obsandbox_validate_certs|d(false)|bool }}"
+#     method: POST
+#     body_format: form-urlencoded
+#     body:
+#     - [ client_id, microcks-serviceaccount ]
+#     - [ client_secret, "{{ _microcks_client_secret }}" ]
+#     - [ grant_type, client_credentials ]
+#     return_content: yes
+#     status_code: 200
+#   register: _microcks_serviceaccount_do_auth
+#   retries: 3
+#   delay: 5
+#   until: _microcks_serviceaccount_do_auth.content is defined
+
+# # - debug:
+# #     msg: "CONTENT ---> {{ _microcks_serviceaccount_do_auth.content | from_json }}"
+
+# - set_fact:
+#     _microcks_serviceaccount_token: "{{ _microcks_serviceaccount_do_auth.content | from_json | json_query('access_token') }}"
+
+# - debug:
+#     msg: "TOKEN ---> {{ _microcks_serviceaccount_token }}"
+
+
+
+
+
+
+
+
+
```

Comment:
Too many blank lines. Please use `yamllint` to catch these issues.

Reply:
resolved